ert_shared/status/tracker/evaluator.py

```diff
@@ -116,13 +112,6 @@ def _drain_monitor(self):
             except (ConnectionRefusedError, ClientError) as e:
```

Comment:
Should we decrease the `_next_ensemble_evaluator_wait_time` just above here, now that this sleep does not affect the total time we search for a new evaluator?

Reply:
Between two iterations, you also need to allow for the EE server to wind down. After termination, the EE server will eventually shut down, but I'm not sure it's instant. So we're sidestepping some synchronization problems with this sleep.

Reply:
True, I forgot about the shutdown time. Then never mind about this!